From ce47c80f3effbdf9264aa22f15fc81f55a45e2e0 Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
Date: Thu, 14 May 2020 13:08:33 +0700
Subject: [PATCH] arm64: dts: renesas: r8a774e1-hihope-rzg2h: remove lossy,
 move CMA and MMP area to lossy area to fix memory size

Lossy can't work with ECC, so we remove the lossy area.
Also move up CMA and MMP area to fix the memory block for data area.

Signed-off-by: Huynh Thanh Hung <hung.huynh.wz@renesas.com>
---
 .../arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts | 12 +++---------
 1 file changed, 3 insertions(+), 9 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts b/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts
index f2ddff4bc5a3..55df02c66ee9 100644
--- a/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts
@@ -29,17 +29,11 @@
 		#size-cells = <2>;
 		ranges;
 
-		/* device specific region for Lossy Decompression */
-		lossy_decompress: linux,lossy_decompress@54000000 {
-			no-map;
-			reg = <0x00000000 0x54000000 0x0 0x03000000>;
-		};
-
 		/* global autoconfigured region for contiguous allocations */
 		global_cma: linux,cma@58000000 {
 			compatible = "shared-dma-pool";
 			reusable;
-			reg = <0x00000000 0x58000000 0x0 0x20000000>;
+			reg = <0x00000000 0x54000000 0x0 0x20000000>;
 			linux,cma-default;
 		};
 
@@ -47,13 +41,13 @@
 		mmp_reserved: linux,multimedia@70000000 {
 			compatible = "shared-dma-pool";
 			reusable;
-			reg = <0x00000000 0x78000000 0x0 0x10000000>;
+			reg = <0x00000000 0x74000000 0x0 0x10000000>;
 		};
 	};
 
 	mmngr {
 		compatible = "renesas,mmngr";
-		memory-region = <&mmp_reserved>, <&lossy_decompress>;
+		memory-region = <&mmp_reserved>;
 	};
 
 	mmngrbuf {
-- 
2.17.1

